basic-examples/parallel/README.md

```diff
@@ -2,56 +2,109 @@
 
 ## Workflow Description
 
-By default, multiple steps in a workflow will be run in parallel if there is no data dependency between the steps. 
+By default, multiple steps in a workflow will be run in parallel if there is no data
+dependency between the steps. 
 
-This workflow runs a metadata collection plugin step and an example plugin step in parallel. All steps are run via the default deployer (defined in `config.yaml`), and their success outputs are reported.
+This workflow runs a metadata collection plugin step and an example plugin step in
+parallel. All steps are run via the default deployer (defined in `config.yaml`), and
+their success outputs are reported.
```

Comment:
Since we're not providing a `config.yaml`, you probably want to omit or amend this last sentence.